granulate_utils/linux/cgroups/base_cgroup.py

```diff
@@ -0,0 +1,96 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from enum import Enum
+from pathlib import Path
+from typing import List
+
+from granulate_utils.linux.cgroups.exceptions import MissingCgroup, MissingController, SkippedCgroup, UnsupportedCgroup
+
+PID_CGROUPS = Path("/proc/self/cgroup")
+CGROUP_PARENT_PATH = Path("/sys/fs/cgroup")
```

Comment:
CGROUPFS is the right name here (the cgroup fs is mounted on `/sys/fs/cgroup` and is possibly mounted on other directories, but it's okay to support only this path for MVP)